test/embedding/embedding.c

```diff
@@ -69,6 +69,14 @@ int main()
         fflush(stdout);
     }
 
+    {
+        // Same as above but using `@cfunction`
+        double (*sqrt_ptr)(double) = jl_unbox_voidpointer(jl_eval_string("@cfunction(sqrt, Float64, (Float64,))"));
+        double retDouble = (*sqrt_ptr)(2.0);
```

Comment:
The `*` here is ignored, so I think it is clearer to exclude it (and less likely for someone to accidentally write the wrong type and have the `*` be accidentally doing something functional)
```suggestion
        double retDouble = sqrt_ptr(2.0);
```

Reply:
Thanks, fixed.